examples/go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/pulumi/pulumi-xyz/examples
-
-go 1.16
+module github.com/pulumiverse/pulumi-acme/examples
+
+go 1.16

```

Comment:
With everything Pulumi, we have Go 1.18 as a default these days. Can you bump the version?